templates/admin/parts/wpa-promotion-widget-sidebar.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+// Security Check
+defined( 'ABSPATH' ) || die();
+?>
+
+<div class="wpa-promotion-content">
+    <h2>WPA Pro Addon</h2>
+    <ul>
+        <li>Integrate your button with contact from 7</li>
+        <li>Receive message directly in your WhatsApp</li>
+        <li>Get priority support and regular updates</li>
+        <li>Early bird discount going on, save up to 40%</li>
+    </ul>
+    <a href="#">Get Now</a>
```

Comment:
`href="<?php echo esc_attr( 'https://wcpress.net/wc-call-for-price-woocommerce-plugin/' ); ?>"`
You have to put a link here with target="_blank"